inst/gates/NK.RM.tsv

```diff
@@ -0,0 +1,11 @@
+levels	use_as	name
+level1	negative	Erythrocyte.RM
+level1	negative	Epithelial.RM
+level1	negative	Stromal.RM
+level1	negative	AvEp.RM
+level1	positive	pDC.RM
+level1	positive	Bcell.RM
```

Comment:
effectively same series of questions as the Myeloid.RM model

Reply:
See above.